Shared/Views/EntryView.swift

```diff
@@ -135,13 +135,13 @@ struct EntryView: View {
                 }
                 
                 if let milestone = campaign.nextMilestone {
-                    MilestoneView(title: "Next milestone", data: campaign, milestone: milestone, showFullCurrencySymbol: showFullCurrencySymbol, showMilestonePercentage: showMilestonePercentage, fillColor: fillColor)
+                    MilestoneView(title: "Upcoming milestones", data: campaign, milestone: milestone, showFullCurrencySymbol: showFullCurrencySymbol, showMilestonePercentage: showMilestonePercentage, fillColor: fillColor)
```

Comment:
Arbitrary: We still refer to 'next milestone' in other places. Do we want to align that to 'upcoming milestones' everywhere.